pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -116,22 +116,36 @@ func LowNodeUtilization(ctx context.Context, client clientset.Interface, strateg
 
 	// log message in one line
 	keysAndValues := []interface{}{
-		"CPU", int64(float64(thresholds[v1.ResourceCPU])),
-		"Mem", int64(float64(thresholds[v1.ResourceMemory])),
-		"Pods", int64(float64(thresholds[v1.ResourcePods])),
+		"CPU", thresholds[v1.ResourceCPU],
+		"Mem", thresholds[v1.ResourceMemory],
+		"Pods", thresholds[v1.ResourcePods],
 	}
 	for name := range thresholds {
 		if !isBasicResource(name) {
 			keysAndValues = append(keysAndValues, string(name), int64(float64(thresholds[name])))
 		}
 	}
 	klog.V(1).InfoS("Criteria for a node under utilization", keysAndValues...)
+	klog.V(1).InfoS("Number of underutilized nodes", "totalNumber", len(lowNodes))
+
+	// log message in one line
+	keysAndValues = []interface{}{
```

Comment:
this is overwriting the `keysAndValues` that was initialized above, but nothing was ever done with it. Is this right?

Reply:
The previous value is printed on line 128:
```
klog.V(1).InfoS("Criteria for a node under utilization", keysAndValues...)
```

Unless I misunderstood the question.

Reply:
Ah, so `keysAndValues` is only used for logging. That sounds familiar yeah